### PR TITLE
Add emborg 1.32.0

### LINKED ISSUE #5322: Add emborg to borg package
### Software Name

emborg

### Brief Description

Emborg is a wrapper similar to borgmatic for making configuration of borg backup jobs simpler.

### Website

https://emborg.readthedocs.io/en/latest/

### Documentation

https://emborg.readthedocs.io/en/latest/

### Build/Installation documentation

https://emborg.readthedocs.io/en/latest/installing.html

### Source code

https://github.com/KenKundert/emborg

### OpenSource License

GPLv3

### Other License

_No response_

### LINKED ISSUE #5340: [Package Request] Update borg to 1.2.1
### Software Name

borgbackup

### Brief Description

borg version 1.2.0 is available as community package.    Thank you!!!
So this is a friendly request for an update. :-)

I just realized that borg **1.2.1** has been releases 6th of June.
There are quite a list of bug fixes and enhancements included. 

### Website

https://github.com/borgbackup/borg/releases

### Documentation

https://github.com/borgbackup/borg/blob/1.2.1/docs/changes.rst#version-121-2022-06-06

### Build/Installation documentation

https://borgbackup.readthedocs.io/en/stable/

### Source code

https://github.com/borgbackup/borg

### OpenSource License

other (please specify below)

### Other License

BSD-3-clause

### PR DESCRIPTION
**Pull request by adhawkins**
## Description

Add emborg to the borg package. emborg is a borg frontend similar to borgmatic. Personally, I find its configuration scheme a little easier to work with when making multiple backups with a similar set of base criteria.

https://emborg.readthedocs.io/en/latest/

Also updates borgmatic to v1.6.3
Also updates borg to v1.2.1

Closes #5322, Closes #5340

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)
- [N/A] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Review by stevenliuit — APPROVED** [review]

**Review by stevenliuit — APPROVED** [review]

**Comment by hgy59** [comment]
TODO: update borg to v1.2.1 (#5340)

File: spk/borgbackup/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SPK_NAME = borgbackup
 SPK_VERS = 1.2.0
```
Comment (hgy59) [inline spk/borgbackup/Makefile:2]:
```suggestion
SPK_VERS = 1.2.1
```

Comment (adhawkins) [reply spk/borgbackup/Makefile]: Whoops, sorry. Should have spotted that. Done.

Comment (adhawkins) [reply spk/borgbackup/Makefile]: If we increase SPK_VER, should SPK_REV be set back to zero?

Comment (hgy59) [reply spk/borgbackup/Makefile]: > If we increase SPK_VER, should SPK_REV be set back to one?

No, SPK_REV must never be decreased. It must be incresed to publish new versions to the syno repo.

Comment (adhawkins) [reply spk/borgbackup/Makefile]: Understood, thanks for the clarification.

Comment (hgy59) [reply spk/borgbackup/Makefile]: @adhawkins you find all about the Makefile variables in our wiki at https://github.com/SynoCommunity/spksrc/wiki/Makefile-variables

**Comment by hgy59** [comment]
@adhawkins I have installed the latest version.

emborg shows the following output for list:
```
$ emborg list
Configuration directory created: /var/services/homes/hgy/.config/emborg.
Includes example settings files. Edit them to suit your needs.
Search for and replace any fields delimited with ⟪ and ⟫.
Delete any configurations you do not need.
Generally you will use either home or root, but not both.
```

Instead of  ⟪ and ⟫ it shows outlined rectangle symbols. It seems that emborg is not console friendly or does not use UTF-8 encoding. Any idea how to fix this?
```
$ locale
LANG=en_US.utf8
LC_CTYPE="en_US.utf8"
LC_NUMERIC="en_US.utf8"
LC_TIME="en_US.utf8"
LC_COLLATE="en_US.utf8"
LC_MONETARY="en_US.utf8"
LC_MESSAGES="en_US.utf8"
LC_PAPER="en_US.utf8"
LC_NAME="en_US.utf8"
LC_ADDRESS="en_US.utf8"
LC_TELEPHONE="en_US.utf8"
LC_MEASUREMENT="en_US.utf8"
LC_IDENTIFICATION="en_US.utf8"
LC_ALL=en_US.utf8
```

**Comment by adhawkins** [comment]
> @adhawkins I have installed the latest version.
> Instead of ⟪ and ⟫ it shows outlined rectangle symbols. It seems that emborg is not console friendly or does not use UTF-8 encoding. Any idea how to fix this?

I see the same with emborg no matter where I install it. However, those characters do appear in the default config file.

Perhaps report this in emborg itself?

**Pull request merged**